recipes/pyflyby/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyflyby" %}
+{% set version = "1.6.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5bebd9fe0f40d809bbf029e3d0d1bfa7bfe2a94379f5a8f383cb85d94fb9a88c
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - "py=pyflyby._py:py_main\npy3=pyflyby._py:py_main"
```

Comment:
This should not have a newline in it

Reply:
CC @marcelotrevisani

Reply:
Did you generated that with grayskull?

Reply:
Their ``setup.py`` is not correct
https://github.com/deshaw/pyflyby/blob/c0448af54d8db8c772da59a124ca667ec4640259/setup.py#L161
They are adding that ``\n``, I can try to do a workaround for it, but this is a very specific case

Reply:
I see. I wonder if this actually installs correctly. If it doesn't, then it's not something that grayskull should worry about.

Reply:
I just tested a `setup.py` install and it does install the entry-points with that syntax.